core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadWriteLDAPUserStoreManager.java

```diff
@@ -92,6 +94,7 @@
     private static final String RETRY_ATTEMPTS = "RetryAttempts";
     private static final String LDAPBinaryAttributesDescription = "Configure this to define the LDAP binary attributes " +
             "seperated by a space. Ex:mpegVideo mySpecialKey";
+    public static final String UTF_8 = "UTF-8";
```

Comment:
StandardCharsets.UTF_8

Reply:
updated in e1d551ca8cbd3d56b44c5da4cd0dea2fcf90057e